### ISSUE TITLE
LaTeX error file not found after installing

### ISSUE THREAD

**Issue by shinokada** [qdm12/latexdevcontainer#9]
After installation of mutirow, I still get LaTeX error.

The output of `tlmgr list --only-installed`:

![image](https://user-images.githubusercontent.com/147320/96361974-0351e580-1165-11eb-8de4-e403ad8ffaf5.png)

> ! LaTeX Error: File `multirow.sty' not found.

```
\documentclass{article}

\usepackage[margin=1in]{geometry}
\usepackage{multirow}

\title{Tables}
\author{John Doe}
\date{}
...
```

**Comment by marvingfx** [comment]
@shinokada I think this is part of #10. I have a quick and dirty fix in there should you still need one.

**Comment by qdm12** [comment]
Closing for now as this might be solved with the resolution of #10  Thanks @marvingfx
